src/Console/ConfigurationResolver.php

```diff
@@ -314,13 +314,15 @@ private function computeConfigFiles()
         }
 
         $path = $this->path;
-        if (is_file($path) && $dirName = pathinfo($path, PATHINFO_DIRNAME)) {
-            $configDir = $dirName;
-        } elseif ($this->isStdIn || null === $path) {
+
+        if ($this->isStdIn || 0 === count($path)) {
             $configDir = $this->cwd;
-            // path is directory
+        } elseif (1 < count($path)) {
+            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
```

Comment:
yep, just specify the config. i would even rename the file into standard `.php_cs.dist`